### DISCUSSION TITLE
Unable to login

### DISCUSSION THREAD

**Discussion by travislongley** [widewing/ha-toyota-na#62] [General]
I am unable to login. I am in NA and am using the toyota app on my phone. I have verified my credentials so there seems to be an API update causing this. Is anyone working on updating the code or is help needed?

**Comment by BluThunder2k** [comment]
Same here, just downloaded this into HA and my user/pass that is standalone for the car fails to let me in.

**Reply by BluThunder2k** [reply]
I followed these instructions (read the entire thread to get full understanding).    Was able to get past the OTP prompt and get the credentials into the integration.   Works now.

**Reply by travislongley** [reply]
I wasn't even getting the email with the code. It just straight up told me the code was wrong. Look at Issue #49 , Supposedly this uses the Toyota-Na Client Library which was taken down. So not sure what is going on. Obviously they don't want their API Key to be out there, but I thought your authentication to your Toyota account would give API access, but maybe not.
